Added welsh (cy)

IeuanWalker opened the pull request:
#1529

Here is a checklist you should tick through before submitting a pull request: 
 - [x] Implementation is clean
 - [x] Code adheres to the existing coding standards; e.g. no curlies for one-line blocks, no redundant empty lines between methods or code blocks, spaces rather than tabs, etc.
 - [x] No Code Analysis warnings
 - [x] There is proper unit test coverage
 - [x] If the code is copied from StackOverflow (or a blog or OSS) full disclosure is included. That includes required license files and/or file headers explaining where the code came from with proper attribution
 - [x] There are very few or no comments (because comments shouldn't be needed if you write clean code)
 - [x] Xml documentation is added/updated for the addition/change
 - [ ] Your PR is (re)based on top of the latest commits from the `main` branch (more info below)
 - [x] Link to the issue(s) you're fixing from your PR description. Use `fixes #<the issue number>`
 - [ ] Readme is updated if you change an existing feature or add a new one
 - [ ] Run either `build.cmd` or `build.ps1` and ensure there are no test failures

IeuanWalker committed:
update

IeuanWalker committed:
remove tests

IeuanWalker committed:
fix csproj

IeuanWalker commented:
Hi, 2 things I'd like to do before changing this draft PR into a complete PR -

- Is there a way to get an output of all the translations being used by the library?
     - I see some languages have custom code to handle their unique differences - https://github.com/Humanizr/Humanizer/tree/main/src/Humanizer/Localisation
     - So I'd like to send an output of what this library would generate to our translation team and they can verify if everything looks ok as is, or if we'd need to look at doing something similar to what other languages have done
- Is there a standard set of tests you expect?
      - Or do you only expect tests if languages have implemented their own code?

IeuanWalker committed:
nuspec added

IeuanWalker committed:
updated translations

hangy commented:
>     * Is there a way to get an output of all the translations being used by the library?
>       
>       * I see some languages have custom code to handle their unique differences - https://github.com/Humanizr/Humanizer/tree/main/src/Humanizer/Localisation
>       * So I'd like to send an output of what this library would generate to our translation team and they can verify if everything looks ok as is, or if we'd need to look at doing something similar to what other languages have done

Not that I know of. Creating all permutations for some languages with support for NumberOfWords, ByteSize, and so on sounds like a lot of information. I wonder if it might be possible to generate that with reflection, though.

>     * Is there a standard set of tests you expect?
>       
>       * Or do you only expect tests if languages have implemented their own code?

As far as I've seen, it's good practice to add some tests with new translations. You're correct that it might seem unnecessary if there's no custom code, but since the strings are also run through `string.Format` or are potentially concatenated in another way, having a test ensures that a change in the base code doesn't break the specific translation.

Do note that I'm just a random contributor and don't speak in any official capacity for the project. :)

IeuanWalker committed:
Merge branch 'main' into Welsh

IeuanWalker committed:
Updated `Humanizer.nuspec` to inlcude `Humanizer.Core.cy`.

Removed duplicate `Humanizer.Core.ku` from `Humanizer.nuspec`

Created `nuget-build-powershell.ps1` to build the nugets locally

clairernovotny committed:
Merge branch 'main' into Welsh

IeuanWalker commented:
@clairernovotny  @SimonCropp  any chance this can be reviewed?

Currently using a custom nuget in a production application and have a few other projects i want to use this in but don't really want to keep copying the nuget to each solution.

clairernovotny committed:
Merge branch 'main' into Welsh
